src/components/event_merchant/vendor_services.vue

```diff
@@ -265,6 +265,13 @@
 import {mapGetters} from 'vuex'
 export default {
   name: 'VendorServices',
+  data(){
+      return{
+          userName: localStorage.getItem('userName'),
+          city: localStorage.getItem('city'),
+          state: localStorage.getItem('state'),
```

Comment:
Any reason why this is fetched from localstorage instead of the State Management?

Reply:
I am yet to work on vendor endpoint that returns specific data, I only save some data on point or registration to the local storage and later fetch it in this component. It has been removed.